src/App.css

```diff
@@ -0,0 +1,35 @@
+.App {
+  background-color: bisque;
+}
+
+.Header {
+  margin: 0;
+  padding: 1rem;
+  background-color: white;
+  border-style: solid;
+  border-width: 1px;
+}
+
+.NavBar {
+  margin: 0;
+  padding: 1rem;
+  background-color: white;
+  border-style: solid;
+  border-width: 1px;
+}
+
```

Comment:
Just a css suggestion for larger screens. 
To stop the cards stretching all the way across and looking massive
max-width: 1100px;
width: 90%;
margin-inline: auto;

Reply:
On articleList btw